### PR TITLE
homepage revamp

### PR DESCRIPTION
**Pull request by alias01**

### CODE REVIEW

File: src/components/organisms/OrganismHomeCard.vue [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+                query: {
+                    latlng: coordinate,
+                },
+                params: {
+                    location: this.LocDetails.locDetails.locName,
+                },
+            });
+        },
+        onChange(tabNumber) {
+            if (tabNumber == 30) {
```
Comment (arrogant-ant) [inline src/components/organisms/OrganismHomeCard.vue:90]: why tabNumber==30?

File: src/components/organisms/OrganismHomeCard.vue [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+                this.$emit('changed', 'car');
```
Comment (arrogant-ant) [inline src/components/organisms/OrganismHomeCard.vue:91]: maybe we can call it as 'VO'

File: src/components/organisms/OrganismHomeCard.vue [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+            } else {
+                this.$emit('changed', 'spot');
```
Comment (arrogant-ant) [inline src/components/organisms/OrganismHomeCard.vue:93]: maybe we can call it as 'SO' for consistency.

File: src/components/templates/TemplateHomeBanner.vue [hunk 4]
```diff
@@ -1,111 +1,129 @@
...
-        ),
-        url('../../assets/Home-Banner.jpg');
-
-    /* Center and scale the image nicely */
-    background-position: center;
-    background-repeat: no-repeat;
-    background-size: cover;
+<style lang="scss" scoped>
+.image-spot-owner {
+    // background-color: yellow;
```
Comment (arrogant-ant) [inline src/components/templates/TemplateHomeBanner.vue:47]: please remove unwanted comments.

File: src/views/ExampleComponent.vue [hunk 5]
```diff
@@ -1,17 +1,13 @@
...
-            {{ message }}
-        </li>
-    </ul>
+    <OrganismHomeCard></OrganismHomeCard>
 </template>
 
 <script>
+import OrganismHomeCard from '../components/organisms/OrganismHomeCard.vue';
 export default {
     name: 'ExampleComponent',
```
Comment (arrogant-ant) [inline src/views/ExampleComponent.vue:8]: this is a test component right?

File: src/views/PageTemp.vue [hunk 6]
```diff
@@ -1,5 +1,5 @@
 <template>
-    <ExampleComponent :messages="messages"></ExampleComponent>
+    <ExampleComponent></ExampleComponent>
     <!-- <div>
```
Comment (arrogant-ant) [inline src/views/PageTemp.vue:3]: remove unwanted code.

**Review by arrogant-ant — CHANGES_REQUESTED** [review]

File: src/components/organisms/OrganismHomeCard.vue [hunk 7]
```diff
@@ -0,0 +1,155 @@
...
+                        <AtomIcon
+                            :icon="'car-back'"
+                            :size="'is-medium'"
+                        ></AtomIcon>
+                        <span> Car Owner </span>
+                    </div>
+                </template>
+                <template>
+                    <div class="card-main-body">
+                        <h2>Search a spot for the car</h2>
```
Comment (arrogant-ant) [inline src/components/organisms/OrganismHomeCard.vue:16]: Search a spot for *your* car

File: src/components/organisms/OrganismHomeCard.vue [hunk 8]
```diff
@@ -0,0 +1,155 @@
+<template>
+    <div class="home-card">
+        <b-tabs expanded @input="onChange">
+            <b-tab-item>
```
Comment (arrogant-ant) [inline src/components/organisms/OrganismHomeCard.vue:4]: please change the selected tab color to blue, its coming up as purple

**Review by arrogant-ant — CHANGES_REQUESTED** [review]

File: src/components/templates/TemplateHomeBanner.vue (hunk 4 above)
Comment (alias01) [reply src/components/templates/TemplateHomeBanner.vue:47]: done

File: src/views/ExampleComponent.vue (hunk 5 above)
Comment (alias01) [reply src/views/ExampleComponent.vue:8]: yes

File: src/views/PageTemp.vue (hunk 6 above)
Comment (alias01) [reply src/views/PageTemp.vue:3]: test components

File: src/components/organisms/OrganismHomeCard.vue (hunk 7 above)
Comment (alias01) [reply src/components/organisms/OrganismHomeCard.vue:16]: done

File: src/components/organisms/OrganismHomeCard.vue (hunk 3 above)
Comment (alias01) [reply src/components/organisms/OrganismHomeCard.vue:93]: done

File: src/components/organisms/OrganismHomeCard.vue (hunk 2 above)
Comment (alias01) [reply src/components/organisms/OrganismHomeCard.vue:91]: done

File: src/components/organisms/OrganismHomeCard.vue (hunk 1 above)
Comment (alias01) [reply src/components/organisms/OrganismHomeCard.vue:90]: tabNumber is returned by the buefy component

Comment (alias01) [reply src/components/organisms/OrganismHomeCard.vue:90]: It is now changed

**Review by arrogant-ant — APPROVED** [review]

**Pull request merged**